packages/blockchain-wallet-v4-frontend/src/pages/CoinPage/components/CoinHeader/CoinHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import { Image, Text } from 'blockchain-info-components'
```

Comment:
@blockdylanb should we use components from the `blockchain-info-components` or should we implement these in constellation too?

Reply:
Images still come from info components, so I think what you have here is ok for now. Text could possibly come from constellation but I haven't actually used it yet so would need to see if it's working properly.

Reply:
yes, I created a spike PR for the text in constellation but I think we still need to wait for Pedro to have the theme done and then they will revisit the Text component in constellation.

So I think there is a long way until we can get the Text component from constellalation.

Should I leave like this for now?

What I could do is try to revisit  the Text component in the blockchain info component to make it type safe, what you think?

Reply:
@awiermanncasas-bc Given that it'll go away eventually I think you can just leave it as is for now.